dowhy/causal_estimators/econml.py

```diff
@@ -91,14 +162,19 @@ def _get_econml_class_object(self, module_method_name, *args, **kwargs):
             )
         return estimator_class
 
-    def _estimate_effect(self):
+    def estimate_effect(
```

Comment:
data: dataframe containing the data on which treatment effect is to be estimated.
treatment_value: value of the treatment variable for which the effect is to be estimated.
control_value: value of the treatment variable that denotes its absence (usually 0)
target_units: The units for which the treatment effect should be estimated. It can be a DataFrame that contains values of the effect_modifiers and effect will be estimated only for this new data. It can also be a lambda function that can be used as an index for the data (pandas DataFrame) to select the required rows.